workspace/apps/pidp/src/app/features/faq/pages/mfa-setup/mfa-setup.page.html

```diff
@@ -222,24 +270,23 @@
               aria-label="mfa enter code and url and click finish" />
           </picture>
         </p>
-        <p class="mfa-section">
-          Whether you scanned the QR code or had to enter the code and URL
-          manually, if successful, your account should now be listed in the
-          Authenticator app.
-          <picture class="image">
-            <source
-              type="image/webp"
-              srcset="/assets/images/mfa-setup/image13.webp" />
-            <source
-              type="image/jpg"
-              srcset="/assets/images/mfa-setup/image13.jpg" />
-            <img
-              alt="Your account listed in the Microsoft Authenticator app screenshot"
-              title="Your account listed in the Microsoft Authenticator app screenshot"
-              loading="lazy"
-              aria-label="mfa account added to authenticator successfully" />
-          </picture>
-        </p>
+      </li>
+      <li class="mfa-section">
+        If successful, your account should now be listed in the Authenticator
+        app.
+        <picture class="image">
+          <source
+            type="image/webp"
+            srcset="/assets/images/mfa-setup/image13.webp" />
+          <source
+            type="image/jpg"
+            srcset="/assets/images/mfa-setup/image13.jpg" />
+          <img
+            alt="Your account listed in the Microsoft Authenticator app screenshot"
```

Comment:
```suggestion
            alt="Your account is listed in the Microsoft Authenticator app screenshot"
```
My grammar mistake 😄